tox.ini

```diff
@@ -35,7 +35,7 @@ deps =
 extras =
   notebook-dependencies
 commands =
-  pytest --nbmake --nbmake-timeout=3000 {posargs} docs/
+  pytest --nbmake --nbmake-timeout=3000 {posargs} --ignore-glob=*forging*serverless* docs/
```

Comment:
Will do, will include [#413](https://github.com/Qiskit-Extensions/quantum-serverless/issues/413) as well

Reply:
This tutorial works now. I updated the code, and Paul had also updated the serverless usage. I'm not sure what fixed it, because the error I encountered before implementing *my* fix was different than before.

Anyway, I'm returning this to its original state

Reply:
Haha, this is great ! :grin:

Reply:
https://github.com/Qiskit-Extensions/quantum-serverless/issues/413#issuecomment-1536645665

Yes! I updated the serverless issue here, and I'll probably recommend they close it soon since this is passing CI

Reply:
This is still broken in 0.6.0 but is fixed in 0.5.0. I think I will move the ignore-glob into the Actions workflow and out of tox, since it only affects the one workflow (I think/hope)

Reply:
Will add a note to the serverless notebook as discussed before

Reply:
We could consider pinning Qiskit nature < 0.6 in the notebook-dependencies only.

Reply:
> We could consider pinning Qiskit nature < 0.6 in the notebook-dependencies only.

I like that. Let's do that instead of ignoring the notebook, and I'll leave the warning in the notebook for now?

Reply:
That sounds perfect to me (as long as CI passes :joy:).  In particular, I think it makes sense to keep the warning in the notebook, in case people are using the tutorial in a setting where they did not install using `[notebook-dependencies]`.